auth/src/main/res/values/config.xml

```diff
@@ -1,5 +1,7 @@
-<resources xmlns:tools="http://schemas.android.com/tools"
-           tools:ignore="ResourceName">
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="ResourceName">
+    <!-- Get access to the google-services project id. -->
+    <item name="project_id" type="string"/>
```

Comment:
What does adding this `item` tag do?

Reply:
Oh boy, forgot about that. Time to remove my hacks and write some docs! 😉